### ISSUE TITLE
Win11: NVDA not reading additional controls in the Win+H voice typing dialog

### ISSUE THREAD

**Issue by SMTestGH** [nvaccess/nvda#12760]
### Steps to reproduce:
1. Press Win+S to open Search so an edit field becomes available
2. Press Alt+H to bring up voice typing dialog
3. Press Win+Alt+H to set focus to Microphone button
4. Tab forward to other controls in dialog (i.e. learn more, settings)

### Actual behavior:
NVDA does not read settings/learn more when a user tabs to them.

### Expected behavior:
NVDA should read the controls when focus lands on them. Narrator does not exhibit the same problem.

#### NVDA installed/portable/running from source:
Installed/Portable

#### NVDA version:
2021.1

#### Windows version:
Windows 11 insider builds

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
N/A

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by josephsl** [comment]
Hi, it does announce these controls on my system (I can confirm this with Windows App Essentials add-on disabled, too). I advise trying with NVDA 2021.2 beta 1 which provides much better support for Windows 11. Thanks.

**Comment by josephsl** [comment]
Hi,

Another key question: exactly which Windows 11 preview build are you using? If you haven't, please try after installing build 22000.160 (KB5005189).

**Comment by seanbudd** [comment]
I cannot reproduce this with NVDA 2021.2 and Windows 11 `22000.194`
